lib/diagnostic.rb

```diff
@@ -5,44 +5,87 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    ar = ['one','two','three','four','five']
+      ar.max_by(&:length) # => "three"
```

Comment:
You need to have a comparison so remember to use the spaceship operator.

`list.max { |a, b| a.length <=> b.length }`